packages/main/src/index.ts

```diff
@@ -97,6 +98,9 @@ if (import.meta.env.PROD) {
 }
 
 let tray: Tray | null = null;
+declare global {
+  let configurationRegistry: ConfigurationRegistry;
+}
```

Comment:
I still think it should be like for tray, no need to use global
else it's looking very good

Reply:
I tried without global, and made every change you suggested. It couldn't find it in the main window and gave this error.
![image](https://user-images.githubusercontent.com/277927/197798432-fa263218-5850-4f36-a925-3d4aa935d64e.png)

Reply:
Would you be fine merging as-is and leaving this for future troubleshooting/optimization by someone who knows what they're doing more than I do?

Reply:
I'll provide you more in-depth review tomorrow (or provide a patch to this PR)

Reply:
@dylanmtaylor I've provided a PR to your branch
https://github.com/dylanmtaylor/podman-desktop/pull/4

And now I'm understanding why you had the undefined error. I thought you were using the object in the same file.

Reply:
I've merged that in